ERCS/erc-7540.md

```diff
@@ -69,11 +69,11 @@ After submission, Requests go through Pending, Claimable, and Claimed stages. An
 |-------------|---------------------------------|-----------|
 | Pending     | `requestDeposit(assets, controller, owner)` | `asset.transferFrom(owner, vault, assets)`; `pendingDepositRequest[controller] += assets` |
 | Claimable   |                                 | *Internal Request fulfillment*:  `pendingDepositRequest[controller] -= assets`; `claimableDepositRequest[controller] += assets` |
-| Claimed     | `deposit(assets, receiver)`      | `claimableDepositRequest[controller] -= assets`; `vault.balanceOf[receiver] += shares` |
+| Claimed     | `deposit(assets, receiver, controller)` | `claimableDepositRequest[controller] -= assets`; `vault.balanceOf[receiver] += shares` |
 
 Note that `maxDeposit` increases and decreases in sync with `claimableDepositRequest`.
 
-Requests MUST NOT skip or otherwise short-circuit the Claim state. In other words, to initiate and claim a Request, a user MUST call both request* and the corresponding claim* function separately, even in the same block. Vaults MUST NOT "push" tokens onto the user after a Request, users MUST "pull" the tokens via the Claim function.
+Requests MUST NOT skip or otherwise short-circuit the Claim state. In other words, to initiate and claim a Request, a user MUST call both request* and the corresponding Claim function separately, even in the same block. Vaults MUST NOT "push" tokens onto the user after a Request, users MUST "pull" the tokens via the Claim function.
```

Comment:
As we define "Lowercase claim always describes the verb action of calling a Claim function." above, I think the old phrasing here is fine.

Reply:
There are no functions that start with the word "claim" though, so the claim* phrasing is a bit weird. You could say "a user MUST call both request* and claim separately" or "a user MUST call both request* and the corresponding Claim function separately"

Reply:
Hm I see your point! Let's keep it as you proposed then.